sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -1339,7 +1339,7 @@ public DigitalTwinsServiceVersion getServiceVersion() {
      * @return An empty mono.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Void> publishTelemetry(String digitalTwinId, String payload) {
+    public Mono<Void> publishTelemetry(String digitalTwinId, Object payload) {
```

Comment:
If we are going to accept raw json strings and send them as-is over the wire, you will need to document that in the javadocs. I don't think users will figure that out otherwise

Reply:
+1 to this.